src/main/java/es/tid/fiware/iot/ac/pap/PoliciesEndpoint.java

```diff
@@ -58,7 +58,7 @@ public Response getPolicy(@Tenant String tenant,
 
         Policy p = dao.loadPolicy(tenant, subject, id);
         if (p != null) {
-            return Response.ok(p.getPolicy()).build();
+            return Response.status(204).entity(p.getPolicy()).build();
```

Comment:
This 204 is being returned in getPolicy but the one we wanted to change was deletePolicy (the following method), didn't we?